GetDKAN/dkan-tools
# Remove make directory

With the changes to how we composer works with Drupal, having a special composer.json file in src/make that gets merged in doesn't make sense anymore. 

I made some small improvements to the docs but we should probably expand the instructions on how to deal with composer.json in git, and the workflow for building/making from an existing project, etc.

One additional improvement: a "schema" folder is created in src, to avoid error messages during the symlink steps.

## QA Steps

1. Check out branch locally
2. Create new project folder
3. `dktl init`
4. `dktl get 8.9.1`
5. `dktl make`
6. `dktl composer require drupal/group` (or any other random dependency)
7. Delete docroot and vendor folders
8. Run dktl make
9. Confirm that your custom dependency is still included
